src/qt/optionsmodel.cpp

```diff
@@ -244,7 +245,8 @@ static ProxySetting GetProxySetting(QSettings &settings, const QString &name)
 
 static void SetProxySetting(QSettings &settings, const QString &name, const ProxySetting &ip_port)
 {
-    settings.setValue(name, ip_port.ip + ":" + ip_port.port);
+    QString proxy = ip_port.ip + QLatin1Char(':') + ip_port.port;
+    settings.setValue(name, proxy);
```

Comment:
This still could be one-liner with wrapping second argument in `QString{}` temporary, or this is assumed to have better readability? If so, `const` could be added.

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/313#issuecomment-841686529).